pkg/volume/glusterfs/glusterfs_uidallocator.go

```diff
@@ -0,0 +1,157 @@
+package glusterfs
```

Comment:
This should be probably made public in pkg/util/*, where OpenShift could find it (so we don't need to maintain two copies in two projects).

I'd appreciate if this file was "generic integer allocator" and we did not use terms like UID, GID or port here (also in the file name).

I miss unit tests.

This can be probably implemented in a separate patch.